src/org-manager/pipes/filter-organisations.pipe.ts

```diff
@@ -0,0 +1,23 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { OrganisationVM } from '../models/organisation';
+
+@Pipe({
+  name: 'filterOrganisations'
+})
+export class FilterOrganisationsPipe implements PipeTransform {
+
+  public transform(orgs: OrganisationVM[], searchFilter: string): OrganisationVM[] {
+    if (!orgs) { return []; }
+    if (!searchFilter || searchFilter === '') { return orgs; }
+    searchFilter = searchFilter.toLowerCase();
+    return orgs.filter((org: OrganisationVM) => {
```

Comment:
was wondering if this bit could be made a bit more readable

Reply:
Please re-review